### DISCUSSION TITLE
Xwidget webkit cannot work under WLS2

### DISCUSSION THREAD

**Discussion by Yanbo** [microsoft/WSL#6506] [General]
# Environment
WLS2, i3wm, emacs, win10
```none
Windows build number: [run `cmd.exe /c ver`]
Your Distribution version: [On Debian or Ubuntu run `lsb_release -r` in WSL]
Whether the issue is on WSL 2 and/or WSL 1: [run `cat /proc/version` in WSL]
```

# Steps to reproduce

I was trying to use xwidget webkit working in emacs under WSL2, but I cannot accesss it 
And I have error output from terminal 

** (emacs:83): WARNING **: 08:58:58.816: GApplication is required for xdg-desktop-portal access in the WebKit sandbox. Actions that require xdg-desktop-portal will be broken.
Unable to init server: Could not connect: Connection refused

(WebKitWebProcess:2): Gtk-WARNING **: 08:58:58.899: cannot open display: 192.168.16.1:0

**WSL logs**: 

#  Expected behavior

# Actual behavior

**Comment by yonta** [comment]
Have you tried to set environment variable, `export WEBKIT_FORCE_SANDBOX=0`?

**Reply by mksonkar** [reply]
this solved the issue for me
